### PR TITLE
feat: Text and icons adjustment

### PR DESCRIPTION
**Pull request by PavloNetrebchuk**
Changes according to this document:
https://www.figma.com/file/iZ56YMjbRMShCCDxqrqRrR/Open-edX-Mobile-App-All-Screens-v2.1?type=design&node-id=9980-2&mode=design&t=m8EjlsKk0aI9cQe8-0
Screenshot 2024-03-21 at 14 29 19
Screenshot 2024-03-21 at 14 30 57
Screenshot 2024-03-21 at 14 31 58

### CODE REVIEW

**Review by volodymyr-chekyrta — DISMISSED** [review]

**Comment by PavloNetrebchuk** [comment]
@sdaitzman 
Screenshot 2024-03-25 at 17 32 25
Done

**Review by k1rill — DISMISSED** [review]

File: core/src/main/java/org/openedx/core/ui/ComposeCommon.kt [hunk 1]
```diff
@@ -1045,40 +1044,48 @@ fun OfflineModeDialog(
...
+                    )
+                }
+                IconButton(
+                    modifier = Modifier.size(20.dp),
+                    onClick = {
+                        onDismissCLick()
+                    }) {
+                    Icon(
+                        modifier = Modifier.size(20.dp),
+                        painter = painterResource(R.drawable.core_ic_close),
```
Comment (omerhabib26) [inline core/src/main/java/org/openedx/core/ui/ComposeCommon.kt:1076]: we can use `imageVector = Icons.Filled.Close,` instead, and remove this drawable.

File: core/src/main/res/drawable/core_ic_close.xml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (omerhabib26) [inline core/src/main/res/drawable/core_ic_close.xml:1]: we can remove this and use `imageVector = Icons.Filled.Close,`

File: core/src/main/res/drawable/core_no_internet_connection.xml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (omerhabib26) [inline core/src/main/res/drawable/core_no_internet_connection.xml:1]: requires auto-format.

File: core/src/main/res/drawable/core_ic_reload.xml [hunk 4]
```diff
@@ -0,0 +1,24 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (omerhabib26) [inline core/src/main/res/drawable/core_ic_reload.xml:1]: 1- can we use white color as a strokeColor to make it more generic?
2- requires auto-format

File: core/src/main/res/drawable/core_ic_offline.xml [hunk 5]
```diff
@@ -0,0 +1,45 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (omerhabib26) [inline core/src/main/res/drawable/core_ic_offline.xml:1]: 1- can we use white color as a strokeColor to make it more generic?
2- requires auto-format

File: course/src/main/res/drawable/course_ic_not_supported_block.xml [hunk 6]
```diff
@@ -3,21 +3,6 @@
     android:height="100dp"
```
Comment (omerhabib26) [inline course/src/main/res/drawable/course_ic_not_supported_block.xml:3]: 1- can we use white color as a strokeColor to make it more generic?
2- requires auto-format

**Review by omerhabib26 — CHANGES_REQUESTED** [review]
Some minor code changes and improvements

**Comment by PavloNetrebchuk** [comment]
@omerhabib26 Changed and formatted👍

**Review by omerhabib26 — APPROVED** [review]

**Review by volodymyr-chekyrta — APPROVED** [review]

**Pull request merged**